resources/user.rb

```diff
@@ -77,3 +83,12 @@ def load_current_value
     new_resource.updated_by_last_action(true)
```

Comment:
This should really go. The execute will handle the resource update so it's not doing anything